### DISCUSSION TITLE
Panel Widths

### DISCUSSION THREAD

**Discussion by hernandito** [coderaiser/cloudcmd#361] [Q&A] [answered]
HI,

I have an iPad Pro, and when using it in portrait mode, the UI shows as **one** panel. I would love it to display **two panels**.

I cant recall what the width resolution of the iPad is, but I would like to slowly lower the threshold between switching from one to two panels. I think it's set somewhere around 1050px.

Where would I make this edit? I tried looking at the cloudcmd.css butI don't think it's in there.

Thank you!

**Comment by coderaiser** [comment]
Hi,

When size of screen is too low `Cloud Commander` uses one panel only. Here is [media query for that](https://github.com/coderaiser/cloudcmd/blob/v15.9.13/css/query.css#L198-L206).

**Comment by hernandito** [comment, marked as answer]
Thank you... it worked!!

I edited the file **_cloudcmd.css_** located in my Docker at:
        `/usr/src/app/dist/`

At around line 568, I changed to:

            `@media only screen and (max-width:655px) {
             .panel {
              width:98%
             }
             .cmd-button#f5,
             .cmd-button#f6,
             .panel-right {
              display:none
             }
            }`

On my iPad, two panels will be OK when viewing by "name-size".

I have made some additional edits to the css file to make the UI a litle less bare. Once I am done getting it looking good, I will share the edits.

![image](https://user-images.githubusercontent.com/787317/148091946-de4ff8c2-eaa9-4394-a209-7c889447cb82.png)

Thank you for your help and for Cloud Commander. I love it.

**Comment by coderaiser** [comment]
It looks very good :)! You can add this media query to [`/css/columns/name-size.css`](https://github.com/coderaiser/cloudcmd/blob/v15.9.13/css/columns/name-size.css), so for this columns two panels was visible.

**Comment by hernandito** [comment]
I am gladd you like it.

I am pasting the css file  for the overall "look". In my Docker, I copied this to:
                `/usr/src/app/dist/cloudcmd.css`

Rename the file from .txt to .css
[cloudcmd.txt](https://github.com/coderaiser/cloudcmd/files/7816785/cloudcmd.txt)

Thank you!

**Reply by coderaiser** [reply]
Could you make a PR with changes you made, so others can benefit  from this modifications :)?
